submodules/users/users.js

```diff
@@ -4849,12 +4887,20 @@ define(function(require) {
 				user = args.user,
 				needVMUpdate = args.needVMUpdate || true,
 				callback = args.callback,
-				oldPresenceId = args.oldPresenceId || undefined,
-				userExtension = args.userExtension;
+				oldPresenceId = args.oldPresenceId || undefined;
 
 			self.usersListVMBoxesUser(user.id, function(vmboxes) {
-				if (vmboxes.length > 0) {
-					if (needVMUpdate) {
+				if (vmboxes.length === 0) {
```

Comment:
You can use `lodash#isEmpty` for this check.
```suggestion
				if (_.isEmpty(vmboxes)) {
```

Reply:
Why would be preferred to use `lodash#isEmpty` for this case?